.appveyor.yml

```diff
@@ -70,5 +70,5 @@ test_script:
 
 on_finish:
   - del C:\projects\devhub.key
-  - codecov
+  - codecov --disable=gcov
```

Comment:
How does this help with long path issues, and why does it only happen now?

Reply:
The codecov tool gcov module was scanning very aggressiv for coverage reports. It also scanned the node_modules folders and caused those errors to appear in the build log. Disabling this made those errors go away and our build pass. I wonder if this was the result of a resent update to the codecov tool - we install it at build time via NPM.